src/i18n.js

```diff
@@ -0,0 +1,33 @@
+import i18n from 'i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+import {initReactI18next} from 'react-i18next'
+import XHR from "i18next-http-backend"
+import EnglishTranslations from './Localization/en/translations.json'
+
+i18n
+.use(XHR)
+.use(LanguageDetector)
+.use(initReactI18next)
+.init({
+    resources: {
+        en: EnglishTranslations
```

Comment:
It would be nice if we can load multiple resource files for one given language. Ideally we have one for core data explorer and one for each use case so the developers don't mess with each other (and data explorer).  Also, it would be nice to only load what we need for a given use case and not load ALL the translations.

One idea: in the function where you dynamically import the right component, import the right translation file(s) as well there.
Another idea: import the translation file inside that self-serve component, so that it gets automatically (and conditionally) imported only when you dynamically import this component.

Reply:
will do in next PR